posts/inside-rust/2020-11-02-Using-rustc_cranelift_codegen.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title: "Using rustc_codegen_cranelift for debug builds"
+author: Joshua Nelson
+team: The Compiler Team <https://www.rust-lang.org/governance/teams/compiler>
+---
+
+## What is `rustc_codegen_cranelift`?
+
+[`rustc_codegen_cranelift`], or just `cg_clif` for short, is a new experimental
+codegen backend for the Rust compiler. The existing backend is LLVM, which very
+good at producing fast, highly optimized code, but is not very good at
+compiling code quickly. `cg_clif`, which uses the [Cranelift] project, would
+provide a fast backend which greatly improves compile times, at the cost of
+only performing very few optimizations. This is a great fit for debug builds, and the hope is
+that `cg_clif` will eventually be the default backend in debug mode.
+
+## What is the progress of using `rustc_codegen_cranelift` for debug builds?
+
+There has been an [Major Change Proposal][MCP] open for some time for making
```

Comment:
```suggestion
There has been a [Major Change Proposal][MCP] open for some time for making
```

Reply:
Hmm, I say 'an em-c-p', which has a vowel at the start. Do you pronounce it differently?

Reply:
I pronounce it the same. What you wrote renders as:

> There has been an [Major Change Proposal]().

Reply:
oh oops :laughing: